utils/s2n_safety_macros.h

```diff
@@ -90,6 +90,11 @@
  */
 #define RESULT_ENSURE_NE(a, b)                                 __S2N_ENSURE((a) != (b), RESULT_BAIL(S2N_ERR_SAFETY))
 
+/**
+ * Ensures `a` implies `b`, otherwise the function will `RESULT_BAIL` with a `S2N_ERR_SAFETY` error
+ */
+#define RESULT_ENSURE_IMPLIES(a, b)                            __S2N_ENSURE(S2N_IMPLIES((a), (b)), RESULT_BAIL(S2N_ERR_SAFETY))
+
```

Comment:
https://github.com/aws/s2n-tls/blob/f5f103371c87d54e3a872249d13bbd720943691f/utils/s2n_safety_macros.h#L19-L24

Reply:
Removed this change; used `RESULT_ENSURE` at call site.